HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,706 @@
+3. As account owner for an account created by an EVM tool my experience with airdrops on Hedera is identical to that on Ethereum
+4. As a network user I want to be able to accurately query the mirror node for all pending transactions sent to an account
+5. As a network user I want to be able to accurately query the mirror node for all outstanding airdrops sent by an account
+6. As a wallet user I want my wallet to show me all pending transactions to my account
+7. As an explorer user I want my explorer to show me search or view all pending transactions and outstanding  to my accounts
+8. As a DApp developer I want to expose airdrop HAPI functionality and pending transfers state information to my users
+9. As a HAPI developer I want to be able to query the cost of an airdrop transactions in HBAR
+10. As a DApp developer I want to be able to query the cost of an airdrop transactions in gas
+11. As an account that has open auto association slots, I only pay for the fees and rent for used association slots. I no longer pay for unused association slots.
+
+### FAQ
+
+- Q: I am sending a crypto transfer, but I don’t want to pay for auto-association. What can I do?
+    
+    A: Each transaction has a `transactionFee` which represents the maximum transaction fee the user is willing to pay for that transaction. If it is set low enough, then the transaction will fail instead of transferring the token in the event an association was required. The user still has to pay node and network fees for the transaction, but the transfer itself will not go through and the client will not be charged for the crypto transfer itself.
+- Q: If I have `maxAutomaticTokenAssociations` set to `0`, and someone airdrops me an NFT, have I received it?
+
+    A: No. In this case, it is *pending*, and you can only receive it by sending a “claim” transaction. When you claim it, the record stream will contain a claim transaction and a token association record entry and a crypto transfer record entry, so that in the record stream, the association and token transfer are both expressed as they normally would be.
+  
+## Specification
+
+From the user perspective, this HIP introduces the following concepts:
+
+- Supporting an explicit unlimited number of automatic token associations when `maxAutomaticAssociations` = -1
+- Defaulting the maximum automatic associations to -1 (unlimited) for auto-created accounts
+- Associating the token-types automatically when a token instance is sent to an account and a slot is available. Calculating and charging the sender for all appropriate fees.
+- Implement new airdrop HAPIs, `TokenAirdrop`, `TokenCancelAirdrop` used by airdrop sender’s to send airdrops and reclaim pending airdrops, and `TokenClaimAirdrop` used by an account to claim a pending airdrop.
```

Comment:
The concept of a "pending transfer" (i.e., "pending airdrop") is itself a new concept here (and it requires a new store in the state and new record stream objects) and could be mentioned with the 3 related HAPI transaction types.

Reply:
Well the store in state is an implementation detail we wanted to leave for groups that implement the HIP. As best as possible we only want to call out the interface and behaviours in the HIP
For the services repo we will expect this level of detail to be captured in the design doc

Reply:
Hmm.  I see the distinction you're thinking of ... but ... given the above discussion where there's (eventually) going to be _rent_ for these pending transfers it's not just an implementation detail ...

Reply:
`new_pending_airdrops` is captured in the transaction record.
I've added a sentence here though